src/winrospublisher.cpp

```diff
@@ -0,0 +1,5 @@
+#include "winrospublisher.h"
+namespace ros_win_camera
+{
+

```

Comment:
Is this a placeholder?

Reply:
yes. I plan to move the definitions from the header to here..

---
In reply to: [383535343](https://github.com/ms-iot/ros_win_camera/pull/2#discussion_r383535343) [](ancestors = 383535343)